File: go/two-fer/two_fer.go
package twofer

// ShareWith returns the string "One for you, one for me.", replacing "you" with an optional `name` argument
func ShareWith(name string) string {
	if (len(name) == 0) {
		name = "you"
	}

	return "One for " + name + ", one for me."
}

File: typescript/two-fer/two-fer.ts
/**
 * This stub is provided to make it straightforward to get started.
 */

export function twoFer(input?: string): string {
  // ^                 ^   ^ this is called a return type; it's the type of the
  // ^                 ^     value that is returned from this function
  // ^                 ^
  // ^                 parameters go here
  // ^
  // allows the tests to import this function and call it
  // <-- Your code goes here. You may remove all the commentary in this file.

  let name = 'you';

  if (input) name = input;
  
  return `One for ${name}, one for me.`
}
